scripts/main.js

```diff
@@ -0,0 +1,112 @@
+
+const Book = (() => {
+  'user strict';
+
+  class BookObj {
```

Comment:
This code looks great, you don't need this  ***BookObj*** class until you're creating a new book, will look much cleaner having the book class outside the function.